### PR TITLE
Update documentation to new format

### PR DESCRIPTION
**Pull request by ganto**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -1,178 +1,372 @@
...
+# restart).
 slapd_default_services: [ 'ldap:///', 'ldaps:///', 'ldapi:///' ]
 
 
-# ---- Network access to OpenLDAP server ----
+# .. Network access to OpenLDAP server [[[1
+#
+# ---------------------------------
+# Network access to OpenLDAP server
+# ---------------------------------
```
Comment (ypid) [inline defaults/main.yml]: The DebOps project normally uses this indention style:

```YAML
# -------------------------------------
#   Network access to OpenLDAP server
# -------------------------------------
```

Comment (ganto) [inline defaults/main.yml]: You're right, I completely missed that here. Will fix it.

File: docs/getting-started.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+         ferm__dependent_rules:
+           - '{{ slapd_ferm_dependent_rules }}'
+
+       - role: debops.tcpwrappers
+         tags: [ 'role::tcpwrappers' ]
+         tcpwrappers_dependent_allow:
+           - '{{ slapd_tcpwrappers_dependent_allow }}'
+
+       - role: debops.slapd
+         tags: [ 'role::slapd' ]
```
Comment (ypid) [inline docs/getting-started.rst]: Want to try out: https://github.com/debops/docs/issues/145 ?

File: docs/introduction.rst [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+This role installs and manages ``slapd``, OpenLDAP server. It will
+automatically generate and store secure passwords for administrator
+accounts and install useful scripts.
+
+
+Installation
+~~~~~~~~~~~~
+
+This role requires at least Ansible ``v1.7``. To install it, run::
```
Comment (ypid) [inline docs/introduction.rst]: https://github.com/debops/ansible-slapd/blob/master/meta/main.yml#L12 suggests that at least 1.8 is required.

**Comment by ypid** [comment]
Looks good, thanks for working on the docs :+1: :wink:

**Pull request merged**